langkit/templates/pkg_implementation_body_ada.mako

```diff
@@ -170,6 +170,23 @@ package body ${ada_lib_name}.Implementation is
    --  If E is known, return its unique Id from State. Otherwise, assign it a
    --  new unique Id and return it.
 
+   pragma Warnings (Off, "referenced");
```

Comment:
If possible, I prefer to avoid these Warnings pragma. What about moving the subprogram spec to the package spec? This makes sense regardless of the Warnings issue anyway IMO.